keyboards/leafcutterlabs/bigKNOBv2/bigKNOBv2.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_1x5( \
+    k01, k02, k03, k04, k05 \
+  ) { \
+    { k01, k02, k03, k04, k05 } \
+}
+
+#define LAYOUT LAYOUT_ortho_1x5 
```

Comment:
I'd prefer one or the other of `LAYOUT` and `LAYOUT_ortho_1x5` be used here, but not both. So either:


```suggestion
#define LAYOUT( \
    k01, k02, k03, k04, k05 \
  ) { \
    { k01, k02, k03, k04, k05 } \
}
```

or

```suggestion
#define LAYOUT_ortho_1x5( \
    k01, k02, k03, k04, k05 \
  ) { \
    { k01, k02, k03, k04, k05 } \
}
```

Whichever you use will also need to be used in:

- https://github.com/qmk/qmk_firmware/blob/8caef9f0c58c69d22fbc72206b4cfe44564601e4/keyboards/leafcutterlabs/bigKNOBv2/info.json#L8
- https://github.com/qmk/qmk_firmware/blob/8caef9f0c58c69d22fbc72206b4cfe44564601e4/keyboards/leafcutterlabs/bigKNOBv2/keymaps/default/keymap.c#L37
- https://github.com/qmk/qmk_firmware/blob/8caef9f0c58c69d22fbc72206b4cfe44564601e4/keyboards/leafcutterlabs/bigKNOBv2/keymaps/media/keymap.c#L37